lasagne/layers/recurrent.py

```diff
@@ -1359,11 +1341,8 @@ def step(input_n, hid_previous, W_hid_stacked, W_in_stacked,
             hid = (1 - updategate)*hid_previous + updategate*hidden_update
             return hid
 
-        def step_masked(input_n, mask_n, hid_previous, W_hid_stacked,
-                        W_in_stacked, b_stacked):
-
-            hid = step(input_n, hid_previous, W_hid_stacked, W_in_stacked,
-                       b_stacked)
+        def step_masked(input_n, mask_n, hid_previous, *args):
+            hid = step(input_n, hid_previous, args)
```

Comment:
It works because you're not using it, but you should pass `*args` instead of `args` (just in case the original step function decides to use `*args` in future).